app/services/user.js

```diff
@@ -1,25 +1,14 @@
-const bcrypt = require('bcrypt');
-const { Sequelize, ...db } = require('../models');
+const models = require('../models');
 const errors = require('../errors');
 const logger = require('../logger');
+const toEncrypt = require('../helpers/encrypt');
 
-const { Op } = Sequelize;
-
-const toEncrypt = data => {
-  const saltRounds = 5;
-  const salt = bcrypt.genSaltSync(saltRounds);
-  const hash = bcrypt.hashSync(data, salt);
-  return hash;
-};
+const { User } = models;
 
 exports.verifyUniqueEmail = async email => {
```

Comment:
It would be better to call this service with a more generic name, perhaps `findUserByEmail`.